### ISSUE TITLE
Concept for a logo/icon

### ISSUE THREAD

**Issue by numanair** [dheijl/swyh-rs#37]
Hello! After using this tool for a bit I thought maybe it should have an icon. Here are some variations on an idea for an icon.
![image](https://user-images.githubusercontent.com/8195699/111257523-81150280-85d8-11eb-952a-5288dc417393.png)
![image](https://user-images.githubusercontent.com/8195699/111258488-53c95400-85da-11eb-8955-e0c21778d71d.png)
![image](https://user-images.githubusercontent.com/8195699/111259051-68f2b280-85db-11eb-8841-31fc201b5938.png)

**Comment by dheijl** [comment]
Thanks, this very much appreciated as my graphical skills are non-existant.

**Comment by dheijl** [comment]
I added some color to your icon, and added it to the exe so that it is visible in Windows Explorer.
But not (yet) visible in the app window title bar for the moment.

![image](https://user-images.githubusercontent.com/2384545/111311677-c9770380-865e-11eb-9f65-74a00d60be4b.png)

**Comment by dheijl** [comment]
And on Windows it's now also in the window title-bar:

![Knipsel](https://user-images.githubusercontent.com/2384545/111323236-f7ae1080-8669-11eb-92d8-ae14bfa1da15.PNG)

**Comment by numanair** [comment]
Sweet. For those small sizes, I wonder if modifying this logo or making an alternate logo is within OS design guidelines?
Here's a zip file with SVGs of the color logo, plus two note-only versions. The only difference is the inner corner. If you have any ideas for improvements I am happy to do them.
[swyh-rs logo svg.zip](https://github.com/dheijl/swyh-rs/files/6176576/swyh-rs.logo.svg.zip)

**Comment by dheijl** [comment]
Thanks! I had resized the color logo to 16x16, 32x32, 64x64 and 128x128 for the Windows .ico resource file, according to MS recommendations. But at least for the very small icon size like in the title bar an alternative would definitely be better!

**Comment by dheijl** [comment]
I changed the smaller icon sizes (16x16, 32x32, 48x48 and 64x64) to the note-only logo in the .ico resource file. I don't know if it is according to guidelines but it seems to be working....

**Comment by numanair** [comment]
After seeing the small version I have tried to better align it to the pixel grid. It should help to make it look sharper.

I think using just the note all around is fine as it is more readable at those small sizes. The only guidelines I found recommended making large-scale 3D icons 2D at small sizes.

In case there is a desire to use the RS in other places I have modified it to match the note-only icon better. There is a taller version and a short version. I think I prefer the tall version in part because it looks less like "AS" at small sizes.
[swyh-rs logo 16x16.zip](https://github.com/dheijl/swyh-rs/files/6178654/swyh-rs.logo.16x16.zip)

Oh, and I meant for the notes to be a single note. I just included both because I hadn't decided if the rounded corner was better.
